pkg/validate/validate.go

```diff
@@ -171,12 +172,26 @@ func IP(ip string) error {
 	return nil
 }
 
-// Interface validates if a string is a valid network interface
-func Interface(iface string) error {
-	if _, err := net.InterfaceByName(iface); err != nil {
-		return fmt.Errorf("%s is not a valid network interface: %s", iface, err)
+// LibvirtInterface validates if an interface is a valid network interface on a libvirt host
+func LibvirtInterface(libvirtURI, ifaceName string) error {
```

Comment:
I think this should be in libvirt specific package.

Reply:
Thanks, good point, I've refactored this a bit to only load those validations with baremetal tag - does that work for you?